server/models/articles.js

```diff
@@ -123,6 +127,32 @@ const getArticleModel = (sequelize, DataTypes) => {
       onUpdate: "CASCADE"
     });
   };
+  Article.fetchArticles = function(options) {
+    const { pageLimit, offset } = pagination(options.query);
+    return this.findAll({
+      order: [["createdAt", "DESC"]],
+      where: options.whereConditions,
+      /* TODO: Add join for ArticleTags */
```

Comment:
The tags have been implemented so please add the tags. Thanks 😄

Reply:
he has not finished his tasks

Reply:
who is he ?

Reply:
Kingsley